site/recent-changes.md

```diff
@@ -1,8 +1,8 @@
-# Recent Changes to the Oracle WebLogic Server Kubernetes Operator
+# Recent changes to the Oracle WebLogic Server Kubernetes Operator
 
-This page tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
+This document tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
 
-| Date | Introduces Backward Incompatibilities | Change |
+| Date | Introduces backward incompatibilities | Change |
 | --- | --- | --- |
 | March 20, 2018 | yes | See [Name Changes](name-changes.md).  Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains.
```

Comment:
If we are removing the name-changes.md this reference should be removed from the comment.

Reply:
True, but we didn't remove name-changes.md. Tom suggested it. He asked for Loren's opinion (so I added Loren as a reviewer), and, Ryan already approved and merged my edits in that file.